### PR TITLE
Script to cleanup old VMs from a project

### PR DESCRIPTION
**Pull request by fejta**
Delete all instances starting with tmp created more than 5 days ago:

```
./clean_project.py --project=kubernetes-jenkins-pull --days=5 --filter=name:'tmp*'
```

### CODE REVIEW

**Comment by fejta** [comment]
Fixes https://github.com/kubernetes/kubernetes/issues/30009

**Comment by spxtr** [comment]
Related: #52

File: jenkins/clean_project.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+def DeleteInstances(project, zones, fake):
+  err = 0
+  for zone, instances in zones.items():
+    base = [
+        'gcloud', 'compute', '-q',
+        'instances', 'delete',
+        '--project=%s' % project,
+        '--zone=%s' % zone
+    ]
+    if fake:
```
Comment (apelisse) [inline jenkins/clean_project.py]: Could you send an actual fake?

```
def DeleteInstances(project, zone, executer = subprocess.call)
```

Comment (fejta) [inline jenkins/clean_project.py]: Hmm... I coded that up but I don't think it makes the code easier to understand or use.

I changed the default behavior to be a dry run, adding a `--delete` flag to really delete.

**Pull request merged**